skll/config.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
                                    'min_feature_count': 'Tuning',
                                    'models': 'Output',
                                    'num_cv_folds': 'Input',
-                                   'objective': 'Tuning',
```

Comment:
Actually, what will happen if someone puts an old style `objective=x` statement in the wrong section? Do we will raise the appropriate error or do we ignore that silently?